EvtWeightTools/src/Data_to_MC_CorrectionInterface_1l_1tau_trigger.cc

```diff
@@ -72,6 +73,12 @@ Data_to_MC_CorrectionInterface_1l_1tau_trigger::~Data_to_MC_CorrectionInterface_
   aux::clearCollection(effTrigger_1m1tau_lepLeg_mc_);
 }
 
+void
+Data_to_MC_CorrectionInterface_1l_1tau_trigger::set_1l_1tau_trigger_sf(correction::Correction::Ref cset)
```

Comment:
I don't see where this function is called.

Reply:
ignoring this for now